### ISSUE TITLE
Hide header

### ISSUE THREAD

**Issue by disrupted** [rcarriga/nvim-notify#34]
Great plugin! I am just getting started to set it up for myself.

One thing I am missing is the ability to hide or customise the whole notification layout, especially the header. Ideally I would want to display notifications exactly like Neogit has it built into their [custom notification module](https://github.com/TimUntersberger/neogit/blob/master/lua/neogit/lib/notification.lua).

Here's an example for notifications sent using `require('neogit.lib.notification').create("...")`
<img width="155" alt="sample notification" src="https://user-images.githubusercontent.com/4771462/140810673-6c3e4a86-fac6-4f47-89f8-bdf1a92a7649.png">

It doesn't display a header (icon, title, time, divider), only the actual message contents.

**Comment by rcarriga** [comment]
I've added configurable rendering so you can use a built-in method or a custom function. See `:h notify-render()`

TL;DR: Add `render = "minimal"` to your setup values
![image](https://user-images.githubusercontent.com/24252670/141534952-bb0cf491-5bb4-473c-9a67-8adb5b23b232.png)
